spaceship.zsh

```diff
@@ -274,6 +275,13 @@ SPACESHIP_EXIT_CODE_SUFFIX="${SPACESHIP_EXIT_CODE_SUFFIX:=" "}"
 SPACESHIP_EXIT_CODE_SYMBOl="${SPACESHIP_EXIT_CODE_SYMBOl:="✘"}"
 SPACESHIP_EXIT_CODE_COLOR="${SPACESHIP_EXIT_CODE_COLOR:="red"}"
 
+# AWS
+SPACESHIP_AWS_SHOW="${SPACESHIP_AWS_SHOW:=true}"
+SPACESHIP_AWS_PREFIX="${SPACESHIP_AWS_PREFIX:="using "}"
```

Comment:
I think `using` will be better for `PREFIX` than `on ` as used in the `REAME.md`.